packages/shared-core/src/helpers/cron.ts

```diff
@@ -0,0 +1,47 @@
+import cronValidate from "cron-validate"
+
+const INPUT_CRON_START = "(Input cron: "
+const ERROR_SWAPS = {
+  "smaller than lower limit": "less than",
+  "bigger than upper limit": "greater than",
+  daysOfMonth: "'days of the month'",
+  daysOfWeek: "'days of the week'",
+  years: "'years'",
+  months: "'months'",
+  hours: "'hours'",
+  minutes: "'minutes'",
+  seconds: "'seconds'",
+}
+
+function improveErrors(errors: string[]): string[] {
+  const finalErrors: string[] = []
+
+  for (let error of errors) {
+    if (error.includes(INPUT_CRON_START)) {
+      error = error.split(INPUT_CRON_START)[0].trim()
+    }
+    for (let [oldErr, newErr] of Object.entries(ERROR_SWAPS)) {
+      if (error.includes(oldErr)) {
+        error = error.replace(new RegExp(oldErr, "g"), newErr)
+      }
+    }
+    finalErrors.push(error)
+  }
+  return finalErrors
+}
```

Comment:
Eep. This concerns me. We're very sensitive to the `cron-validate` library changing their wording slightly and breaking our error messages. Makes me wonder if cron validation is actually complex enough to warrant using a library and munging its error messages.

Reply:
If we lock the library version then this shouldn't be too much of a problem - I doubt we will need to update this often - the error messages themselves are quite hard to read so just wanted to improve them ever so slightly.